src/core/fonts.js

```diff
@@ -722,6 +722,11 @@ var Font = (function FontClosure() {
   function int16(b0, b1) {
     return (b0 << 8) + b1;
   }
+  
+  function signed_int16(b0,b1) {
+    var value = (b0 << 8) + b1;
+    return value & 1<<15 ? value - 0x10000 : value;
```

Comment:
Nit: missing spaces around the `<<` operator (and it might be a good idea to enclose that part in parentheses), i.e. `return value & (1 << 15) ? value - 0x10000 : value;`